types/helpers/helpers.core.d.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * An empty function that can be used, for example, for optional callback.
+ */
+export function noop(): any;
+
+/**
+ * Returns a unique id, sequentially generated from a global variable.
+ * @returns {number}
+ * @function
+ */
+export function uid(): number;
+/**
+ * Returns true if `value` is neither null nor undefined, else returns false.
+ * @param {*} value - The value to test.
+ * @returns {boolean}
+ * @since 2.7.0
+ */
+export function isNullOrUndef(value?: any): value is undefined;
```

Comment:
1. Does it really make sense for the parameter to be optional here? I'd declare it as mandatory.
2. Wouldn't the proper return statement be `value is null | undefined`?